samples/compute/v2/servers/add_remove_security_group.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+require 'vendor/autoload.php';
```

Comment:
can we split this into 2 files, so: `add_security_group.php` and `remove_security_group.php`